README.md

```diff
@@ -125,9 +125,31 @@ Other values are managed by the Vue.js application and are provided in the [`_de
 
 ## Releasing
 
-To publish a new version on the marketplace:
-* Update the version in the main class of this module, and in `config.xml` ifit exists,
-* Create a new release on the [releases page](https://github.com/PrestaShopCorp/pasmarketingwithgoogle/releases) with a tag that matches the version provided in the main class.
+### QA Review
+
+Each version need to be tested by the QA team before being sent to production.
+This step is done once all pull-requests are merged in the base branch (`master` for instance).
+
+To create a pre-release:
+* Check the version in the main class of the module and `config.xml` match the upcoming version, or update the values accordingly,
```

Comment:
Il ne faut pas vérifier le fichier `psxmarketingwithgoogle.php` aussi ?

Reply:
Indeed, thus the part about `the main class of the module`